engine/src/main/java/org/hibernate/search/engine/search/dsl/projection/impl/DefaultSearchProjectionFactoryContext.java

```diff
@@ -45,14 +46,26 @@ public DocumentReferenceProjectionContext documentReference() {
 	public <T> FieldProjectionContext<T> field(String absoluteFieldPath, Class<T> clazz) {
 		Contracts.assertNotNull( clazz, "clazz" );
 
-		return new FieldProjectionContextImpl<>( factory, absoluteFieldPath, clazz );
+		return new FieldProjectionContextImpl<>( factory, absoluteFieldPath, clazz, ProjectionConverter.ENABLED );
 	}
 
 	@Override
 	public FieldProjectionContext<Object> field(String absoluteFieldPath) {
 		return field( absoluteFieldPath, Object.class );
 	}
 
+	@Override
+	public <T> FieldProjectionContext<T> rawField(String absoluteFieldPath, Class<T> clazz) {
+		Contracts.assertNotNull( clazz, "clazz" );
+
+		return new FieldProjectionContextImpl<>( factory, absoluteFieldPath, clazz, ProjectionConverter.DISABLED );
+	}
+
+	@Override
+	public FieldProjectionContext<Object> rawField(String absoluteFieldPath) {
+		return field( absoluteFieldPath, Object.class );
```

Comment:
This seems wrong, it should be `rawField( absoluteFieldPath, Object.class )`. And tests do not fail, so you're missing a test here.

Reply:
You're right! There's a very dangerous typo here!

I've fixed it and added a test for using `rawField` without expressing a return type. 
Pre-exist test used `rawField`, always expressing the return type.

Reply:
Nice idea to have changed the existing test for `.field` too.